### PR TITLE
Add support for creating multiple beds

### LINKED ISSUE #5402: Bulk Create Beds
**Is your feature request related to a problem? Please describe.**
Locations tend to have 5-15 Beds with very similar names, there should be a better UX option to create them together while setting up the facility

**Describe the solution you'd like**
Create Multiple beds together

### PR DESCRIPTION
**Pull request by siddnikh**
- Fixes  #5402
- Needs BE PR merge: https://github.com/coronasafe/care/pull/1275

### CODE REVIEW

**Comment by rithviknishad** [comment]
@siddnikh is this good for testing? mark with need testing label if so.

File: src/Components/Facility/AddBedForm.tsx [hunk 1]
```diff
@@ -205,7 +224,29 @@ export const AddBedForm = (props: BedFormProps) => {
...
                 </div>
-
+                {!bedId && (
+                  <div>
+                    <div>
+                      <LegacyCheckboxField
+                        label="Do you want to make multiple beds?"
+                        checked={multipleBeds}
+                        onClick={() => setMultipleBeds(!multipleBeds)}
+                      />
```
Comment (rithviknishad) [inline src/Components/Facility/AddBedForm.tsx]: Use `CheckBoxFormField` instead. Avoid using legacy components in new pieces of code.

**Review by rithviknishad — CHANGES_REQUESTED** [review]
Other than that LGTM

File: src/Components/Facility/AddBedForm.tsx (hunk 1 above)
Comment (siddnikh) [reply src/Components/Facility/AddBedForm.tsx]: Okay sure, I'll change this.

**Comment by siddnikh** [comment]
> @siddnikh is this good for testing? mark with need testing label if so.

Waiting for backend.

**Comment by gigincg** [comment]
@nihal467 Can you get this tested with the Backend branch?

cc: @rithviknishad In case @nihal467 needs help to run the branch locally

**Comment by nihal467** [comment]
@siddnikh fix the deploy failure

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
@siddnikh 
image

@gigincg I tested in the tunnel provided by Rithvik, there are bugs : 
1. The bed-type heading is missing
2. unable to create multiple beds ( Internal server error )

**Comment by nihal467** [comment]
LGTM, tested through rithvik cloudflare tunnel

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**